qutebrowser/components/adblock.py

```diff
@@ -131,6 +131,37 @@ def filter_request(self, info: interceptor.Request) -> None:
                         .format(info.request_url.host()))
             info.block()
 
+    def _read_hosts_line(self, line: str) -> list:
```

Comment:
This should be `typing.List[str]` instead of just `list` (or `typing.Set`, see below)